nix/sources.json

```diff
@@ -44,7 +44,7 @@
       "homepage": null,
       "owner": "NixOS",
       "repo": "nixpkgs",
-      "rev": "d68537fe60c16405c6673f6a06ca70607e443fb9",
+      "rev": "1882c6b7368fd284ad01b0a5b5601ef136321292",
```

Comment:
This doesn't change the `sha256` so it does nothing. I guess you just edited this by hand instead of using `niv update` :)

Reply:
I fixed this locally and things seemed to get a lot further before I got bored and killed it.